### ISSUE TITLE
Font inclusion warnings

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/mn-native-pdf#91]
I'm seeing these font warnings in mn-native-pdf-nist.

```
WARNING: Font "Cambria Math,normal,700" not found. Substituting with "Cambria Math,normal,400".
WARNING: Font "HanSans,italic,700" not found. Substituting with "HanSans,italic,400".
```

Are these font weights missing? Can we include them in mn2pdf?

P.S. Why is HanSans used in NIST documents?

**Comment by Intelligent2013** [comment]
> Are these font weights missing? Can we include them in mn2pdf?

@ronaldtse I've added it into FOP config file for mn2pdf. But really these fonts weights/styles don't using (I've compared PDF with and without it), looks like it's just FOP warning.
mn2pdf updated.

> P.S. Why is HanSans used in NIST documents?

I've remove HanSans from NIST template. In some old documents there were a few non-printable characters, which don't included in Times. Now there aren't.
xslt in mn-native-pdf-nist updated.
